docs/news/index.qmd

```diff
@@ -9,6 +9,101 @@ format:
 
 This page provides the release notes associated with each release of RStudio and Posit Workbench. Please contact customer support (<a href="mailto:support@posit.co">support@posit.co</a>) for questions about the described changes.
 
+## RStudio 2023.12.0
+
+**"Ocean Storm"**
+
+> Date: 2023-12-15
+
+### New
+
+#### RStudio
+
```

Comment:
@timtmok does @jthomasmock want us to update our note about Copilot? In Desert Sunflower, it mentioned "This feature is currently in public Preview". Should this be updated for Ocean Storm?